features/steps/steps.py

```diff
@@ -246,6 +257,7 @@ def step_impl(context, related, relating, other_entity, condition):
     stmt_to_op = {"is": operator.eq, "is not": operator.ne}
     assert condition in stmt_to_op
     pred = stmt_to_op[condition]
+
```

Comment:
How come this no longer calls the function?

Reply:
That will cause an AssertionError I think, as the 'is' comes from the sentence ".. shall be assigned to {relating} if {other_entity} {condition"} present'
How do you feel about putting the stmt_to_op dictionary into the function call? This way, the sentences are not dependent on the stmt_to_op function, it's called with only one line of code and it's easy to see for someone looking into the code what is happening.  
```
>>> import operator
>>> def get_op(statement, stmt_to_op = {}):
...     assert statement in stmt_to_op
...     return stmt_to_op[statement]
... 
>>> pred = get_op("is", {"is":operator.eq, "is not": operator.ne})
>>> pred
<built-in function eq>
```

Reply:
> How do you feel about putting the stmt_to_op dictionary into the function call?

Strongly against.

**Proposal 1**

Consider calling

> `statement = statement.replace("is", "")`

or sth similar as the first call within `stmt_to_op` function (probably need to account for spaces as well).

Add `"" : operator.eq, # a == b` to the `stmt_to_op` dictionary. In this way, a lot more of cases should be supported.

**Proposal 2**

Add `"is"` and all other permutations to the dictionary `stmt_to_op` as well. Similar result as Proposal 1.

Reply:
> `statement = statement.replace('is','').strip()`

I like this very much. This also extends the function to cover cases like "is equal to" to map to "equal to" and thus make a very versatile function.

> Then we get to the same situation where we started, or do I miss something?

Proposal 2 was just an alternative, I didn't like it either. But would still prefer it over a dictionary parameter to the function - in that case there is no reuse of the logic: "some string" maps to `operator` and we truly are where we started :)

Reply:
No, there would be a need to provide a dictionary parameter every time indeed. 
Proposal 1 will be easy to work with I think, it's implemented in https://github.com/buildingSMART/ifc-gherkin-rules/pull/23/commits/5c15d0e8d0a6e1278ef5b515d5f6cc052b8d511d